Sources/SwiftIDEUtils/SyntaxClassification.swift

```diff
@@ -72,25 +68,27 @@ extension SyntaxClassification {
     case \DeclModifierSyntax.name:
       return (.attribute, false)
     case \ExpressionSegmentSyntax.leftParen:
-      return (.stringInterpolationAnchor, true)
+      return (.none, true)
     case \ExpressionSegmentSyntax.rightParen:
-      return (.stringInterpolationAnchor, true)
+      return (.none, true)
```

Comment:
I think we can just remove these two cases because parentheses are classified as `.none` by default, so there’s nothing to override